pydantic_ai_slim/pydantic_ai/profiles/openai.py

```diff
@@ -38,6 +38,9 @@ class OpenAIModelProfile(ModelProfile):
     openai_system_prompt_role: OpenAISystemPromptRole | None = None
     """The role to use for the system prompt message. If not provided, defaults to `'system'`."""
 
+    openai_supports_web_search: bool = False
```

Comment:
Let's make it clear this is Chat Completions-specific

```suggestion
    openai_chat_supports_web_search: bool = False
```